nginx.conf

```diff
@@ -38,12 +38,12 @@ http {
   			proxy_pass http://$VulnerableApplication:80;
         }
         
-        location /VulnerabilityDefinitions {
+        location ~ /(VulnerabilityDefinitions | scanner) {
```

Comment:
We want to cater both Sast and Dast, so how will we be able to do from /scanner ? are we looking for a new parameter or shall we create 2 different endpoints?

Reply:
It is not possible from just /scanner, can have two endpoints -> /scanner/sast and /scanner (for dast).

Reply:
let's have 2 endpoints `/scanner/sast` and `/scanner/dast` and route /scanner/dast to /scanner for VulnerableApp and for other vulnerable applications let's route to `/scanner/dast`

Reply:
created the two endpoints. Hopefully this looks okay, was trying to avoid duplicating lines in the nginx.conf.